content/src/controller.js

```diff
@@ -1907,10 +1907,12 @@ function sortByDate(a, b) {
 }
 
 function sortByName(a, b) {
-  if (a.name < b.name) {
+  var aName=a.name.toLowerCase();
+  var bName=b.name.toLowerCase();
+  if (aName < bName) {
     return -1;
   }
-  if (a.name > b.name) {
+  else if (aName > bName) {
     return 1;
   }
   return 0;
```

Comment:
It is good for the primary sort order to be case-insensitive.

But when there is a tie, we should fall back to compare things in a case-sensitive fashion so that Foo and foo are not tied.  One should come before the other.